src/Elastic.Markdown/HtmlWriter.cs

```diff
@@ -147,10 +149,10 @@ private async Task<RenderResult> RenderLayout(MarkdownFile markdown, MarkdownDoc
 			Features = DocumentationSet.Configuration.Features,
 			StaticFileContentHashProvider = StaticFileContentHashProvider,
 			ReportIssueUrl = reportUrl,
-			CurrentVersion = legacyPages?.Count > 0 ? legacyPages.ElementAt(0).Version : "9.0+",
+			CurrentVersion = currentBaseVersion,
 			AllVersionsUrl = allVersionsUrl,
 			LegacyPages = legacyPages?.Skip(1).ToArray(),
-			VersionDropdownItems = VersionDrownDownItemViewModel.FromLegacyPageMappings(legacyPages?.Skip(1).ToArray()),
+			VersionDropdownItems = VersionDropDownItemViewModel.FromLegacyPageMappings(legacyPages?.Skip(1).ToArray()),
```

Comment:
I think the `.Skip(1)` needs to be removed here.

Related to @colleenmcginnis's [comment](https://github.com/elastic/docs-builder/pull/1865/files#r2361359957)